src/QMCWaveFunctions/LCAO/SoaLocalizedBasisSet.cpp

```diff
@@ -21,6 +21,58 @@
 
 namespace qmcplusplus
 {
+
+template<class COT, typename ORBT>
+void SoaLocalizedBasisSet<COT, ORBT>::createResource(ResourceCollection& collection) const
+{
+  for (int i = 0; i < LOBasisSet.size(); i++)
+    LOBasisSet[i]->createResource(collection);
+}
+template<class COT, typename ORBT>
+void SoaLocalizedBasisSet<COT, ORBT>::acquireResource(
+    ResourceCollection& collection,
+    const RefVectorWithLeader<SoaBasisSetBase<ORBT>>& basis_list) const
+{
+  auto& loc_bs_leader  = basis_list.template getCastedLeader<SoaLocalizedBasisSet<COT, ORBT>>();
+  auto& atom_bs_leader = loc_bs_leader.LOBasisSet;
+  const int num_ctr    = loc_bs_leader.LOBasisSet.size();
+  for (int i = 0; i < num_ctr; i++)
```

Comment:
could you rename atom_bs_leader to basisset_leader?
could you get rid of num_ctr? Just use basisset_leader.size();
Also update `releaseResource` accordingly.

Reply:
fixed